config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  #get 'root/index'
+  root 'root#index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
```

Comment:
- [OPTIONAL] Good job implementing required changes from the previous reviewer. However, I noticed there's an error at the first load. The root path seems to be having issue. Kindly look into it and fix. Take a look :point_down: 

![error-load](https://user-images.githubusercontent.com/26597432/236146918-7d852031-adca-4ea2-b20e-304c0e9fff62.png)